chat-commands.js

```diff
@@ -2274,9 +2274,11 @@ exports.commands = {
 			target = targets[1].trim();
 			roomId = toId(targets[0]) || room.id;
 		}
+		let targetRoom = Rooms.get(roomId);
+		let addModlogLinks = Config.modloglink && (!hideIps || (Config.roomauthModlogLink && targetRoom && !targetRoom.isPrivate));
 
 		// Let's check the number of lines to retrieve or if it's a word instead
-		if (!target.match('[^0-9]')) {
+		if (!target.match('\D')) {
```

Comment:
This shouldn't be a string anyway, for readability reasons. And I still like `/[^0-9]/`, also for readability reasons (much easier than remembering what `\D` stands for, which by the way should have been `'\\D'` because it's a string).